test/examples/Makefile.am

```diff
@@ -25,24 +25,34 @@ installcheck-local:
 
 if HAVE_EXAMPLES
 
-EXAMPLE_INCLUDE_FLAGS = -I$(abs_top_srcdir)/src -I$(abs_top_builddir)/src
+EXAMPLE_INCLUDE_FLAGS =     \
+	-I$(abs_top_srcdir)/src \
+	-I$(abs_top_builddir)/src
 
-noinst_PROGRAMS = ucp_hello_world uct_hello_world ucx_profiling ucp_client_server
+EXAMPLE_CFLAGS =   \
+	$(BASE_CFLAGS) \
+	$(EXAMPLE_INCLUDE_FLAGS)
```

Comment:
it's better to add includes to CPPFLAGS rather than CFLAGS (CPP == C PreProcessor, this is common to both C++ and C programs)

Reply:
right, it was going to be common for all examples locally built
ok, fixed